GW2EIEvtcParser/ParsedData/CombatData.cs

```diff
@@ -998,6 +998,18 @@ public IReadOnlyList<EffectEvent> GetEffectEventsByTrackingID(long trackingID)
             return new List<EffectEvent>();
         }
 
+        public bool TryGetEffectEndByTrackingId(long trackingID, long time, out long end)
+        {
+            end = 0;
+            EffectEvent endEvent = GetEffectEventsByTrackingID(trackingID).FirstOrDefault(effect => effect.EffectID == 0 && effect.Time >= time);
```

Comment:
I haven't been using it at this point, but it'd be cleaner to have a property IsEnd => EffectID == 0 in EffectEvent.cs and use that instead for better readability